packages/fluentui/accessibility/src/behaviors/Tree/treeItemBehavior.ts

```diff
@@ -5,6 +5,8 @@ import { IS_FOCUSABLE_ATTRIBUTE } from '../../attributes';
 import treeTitleBehavior from './treeTitleBehavior';
 
 /**
+ * @description
+ * Triggers 'performClick' action with 'Spacebar' on 'root', when tree item is selectable and has no subtree. 'Enter' or 'Spacebar' works otherwise.
```

Comment:
To be honest, I have no idea what this sentence mean 

> 'Enter' or 'Spacebar' works otherwise.

Reply:
```suggestion
 * Triggers 'performClick' action with 'Spacebar' on 'root', when tree item is selectable and has no subtree. In other cases 'performClick' is triggered with 'Spacebar' or 'Enter'.
```